diff -p -up swh-plugins-0.4.15/metadata/Makefile.in.build swh-plugins-0.4.15/metadata/Makefile.in
--- swh-plugins-0.4.15/metadata/Makefile.in.build	2006-08-08 17:48:03.000000000 +0200
+++ swh-plugins-0.4.15/metadata/Makefile.in	2009-12-12 14:06:01.000000000 +0100
@@ -358,8 +358,6 @@ swh-plugins.rdf: @top_srcdir@/*.xml
 	./lxml2rdf.pl @top_srcdir@/*.xml > swh-plugins.rdf
 	xmllint -noout swh-plugins.rdf
 
-swh-scales.rdf: scale-points.txt
-	./txt2scale.pl scale-points.txt > swh-scales.rdf
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
